doc/gov/chapters/chapter05.md

```diff
@@ -33,8 +33,19 @@ Being on the voting representative list **doesn't** mean the right to vote. The
 
 <sup>2)</sup>The criteria will be checked and the voting representative list will be updated prior any voting event when the voting process kicks in.
 
-<sup>3)</sup> Individuals who don't present an organisation will not have voting right.
+<sup>3)</sup> Individuals who don't represent an organisation will not have voting rights.
```

Comment:
This superscript item 3 is not referenced

Reply:
@pgoyal01 fixed